EPI/create-VS-from-CAG.py

```diff
@@ -41,9 +45,13 @@ def analyze_cag(input):
     with open(input, "rt") as f:
         model = json.load(f)
 
+        if "metadata" not in model:
```

Comment:
So we can't generate the SIR CAG without metadata then?

Reply:
We're using the top-level metadata to identify the variables and constants properly. I suppose we can use the fact that a node has no incoming edges, but I'd have to have a closer look to see if that works for the simple SIR and double epi models.

Reply:
Let's merge this one and we can address that in a subsequent PR. I'd like to incorporate the 3 VS to the HMI to see how that looks in the near future.

Reply:
Ok, this has been merged. But just to be clear, incorporating the 3 VS we have doesn't depend on this PR correct? We can continue to incorporate the 3 VS we had previously created manually.